playerbot/PlayerbotMgr.cpp

```diff
@@ -9,30 +8,54 @@
 #include "Chat/ChannelMgr.h"
 #include "Social/SocialMgr.h"
 
-
 class LoginQueryHolder;
 class CharacterHandler;
 
 PlayerbotHolder::PlayerbotHolder() : PlayerbotAIBase()
 {
     for (uint32 spellId = 0; spellId < sServerFacade.GetSpellInfoRows(); spellId++)
+    {
         sServerFacade.LookupSpellInfo(spellId);
+    }
 }
 
 PlayerbotHolder::~PlayerbotHolder()
 {
 }
 
+void PlayerbotHolder::ForEachPlayerbot(std::function<void(Player*)> callback) const
+{
+    for (auto& itr : playerBots)
+    {
+        Player* bot = itr.second;
+        if (bot)
+		{
+	    	callback(bot);

```

Comment:
Tabs and whitespaces mixed. 😓

Reply:
I edited it via github, seems like it doesn't work very well